tools/dep_updaters/update-eslint.sh

```diff
@@ -63,3 +72,6 @@ rm -rf ../node_modules/eslint
 
 mv eslint-tmp/node_modules/eslint ../node_modules/eslint
 rm -rf eslint-tmp/
+
+# The last line of the script should always be the NEW_VERSION
+echo "$NEW_VERSION"
```

Comment:
For the above to work, you would need to also update the script. I think the following change would also make the output more human-friendly, which is always nice.
```suggestion
echo "NEW_VERSION=$NEW_VERSION"
```